packages/ui/src/DateUtilities.tsx

```diff
@@ -254,14 +254,25 @@ export function printDateAndTime(
 export function printDateRange(
   start: string,
   end: string,
-  {timezone, showTimezone = true}: {timezone: string; showTimezone?: boolean}
+  {
+    timezone,
+    showTimezone = true,
+    timeOnly,
+  }: {timezone: string; showTimezone?: boolean; timeOnly?: boolean}
 ): string {
   const startDate = printDate(start, {timezone, showTimezone: false});
   const endDate = printDate(end, {timezone, showTimezone: false});
 
   const startTime = printTime(start, {timezone, showTimezone: false});
   const endTime = printTime(end, {timezone, showTimezone});
-  if (startDate === endDate) {
+  if (timeOnly) {
+    if (startDate !== endDate) {
+      console.warn(
+        `printDateRange: printing only time but start and end dates are different: ${start} - ${end}`
+      );
+    }
+    return `${startDate} ${startTime} - ${endTime}`;
```

Comment:
It returns the same thing as the subsequent `if` and still the date?

Reply:
Whoops fixed!